docs/6.Events.md

```diff
@@ -467,6 +467,32 @@ class Module
 }
 ```
 
+Note: since v1.0.1 we have decoupled from zendframework/zend-mvc and as such the event manager isn't reliably available in the service container. If you are not using zend-mvc and still rely on this feature you should inject a shared event manager into your container yourself.
```

Comment:
I would rephrase as:

> Note: since v1.0.1 we have decoupled from zendframework/zend-mvc and as such the shared event manager isn't available in the service container. If you are not using zend-mvc you should not use the shared event manager.